### DISCUSSION TITLE
Samples/Further Description for projects.json and setup.cfg

### DISCUSSION THREAD

**Discussion by devaniranjan** [chaoss/grimoirelab#677] [Q&A] [answered]
I am testing out grimoirelab and I can set up the default system and run it with no issues using docker-compose. I can generate the grimoirelab dashboard. 

I am having some challenges when I try to update the projects.json file and setup.cfg for my own use case. 

Does anyone have example snippets for the following uses that I can look at to fix the issue?

1. Pull git repos from two different orgs/projects [E.g. Kubernetes and PyTorch]
2. Pull up issues by updating the setup.cfg file

Thanks.

**Comment by zhquan** [comment, marked as answer]
Hi @devaniranjan 

Here is an example of `projects.json` with these projects. GrimoireLab cannot pull all repositories from a GitHub organization so you have to add all repositories manually.

```
{
    "Kubernetes": {
        "git": [
            "https://github.com/kubernetes/kubernetes",
            "https://github.com/kubernetes/autoscaler"
        ],
        "github": [
            "https://github.com/kubernetes/kubernetes",
            "https://github.com/kubernetes/autoscaler"
        ]
    },
    "PyTorch": {
        "git": [
            "https://github.com/pytorch/pytorch",
            "https://github.com/pytorch/vision"
        ],
        "github": [
            "https://github.com/pytorch/pytorch",
            "https://github.com/pytorch/vision"
        ]
    }
}
```

Add the `github` section on your `setup.cfg` (more info at https://github.com/chaoss/grimoirelab-sirmordred?tab=readme-ov-file#github-)

```
[github]
raw_index = github_raw
enriched_index = github_enriched
api-token = ["token"]
sleep-for-rate = true
no-archive = true
```

I hope it helps you.

Best,
Quan

**Comment by devaniranjan** [comment]
Thank you! I was able to add many repos from an organization.
